src/surveys.rs

```diff
@@ -0,0 +1,82 @@
+use std::collections::{HashMap, BTreeMap};
+
+use serde::{Serialize, Deserialize};
+
+use crate::UserGroups;
```

Comment:
```suggestion
use std::prelude::*;
```

Et rajoute HashMap dans le prélude